cli/scale.go

```diff
@@ -91,7 +91,12 @@ func runScale(args *docopt.Args, client *controller.Client) error {
 		if err != nil {
 			fmt.Println(commands["scale"].usage)
 		}
-		processes[arg[:i]] = val
+		processType := arg[:i]
+		if _, ok := release.Processes[processType]; ok {
+			processes[processType] = val
+		} else {
+			fmt.Printf("No process type exists with name \"%s\".\n", processType)
```

Comment:
Well, if I run `flynn scale web=1 workr=2` I've obviously made a typo, my intention was to scale both the `web` and `worker` processes so I would like Flynn to tell me that and exit, leaving me to re-run the command correctly to do what I actually wanted. WDYT?